website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,172 @@
+
+For detailed specifications, see the [Deployment Safeguards documentation](https://learn.microsoft.com/azure/aks/deployment-safeguards).
+
+## Pod Readiness SLA for AKS Automatic
+
+![API Server Uptime SLA: 99.95% control plane uptime. Pod Readiness SLA: 99.9% operations under 5 minutes](automatic-slas.png)
+
+Uptime means more than a healthy control plane; it means your applications are actually serving users. The [Pod Readiness SLA](https://www.microsoft.com/licensing/docs/view/Service-Level-Agreements-SLA-for-Online-Services) guarantees that pods reach readiness targets, closing the gap between "the cluster is healthy" and "my app is ready."
+
+- **Faster recovery during failures:** Node failures and scale events trigger remediation so pods return to a ready state within defined thresholds.
+- **Predictable reliability:** Availability planning aligns with measurable guarantees instead of best-effort behavior.
+- **Reduced operational overhead:** Platform automation handles remediation, eliminating manual firefighting during disruptions.
+- **Business continuity at scale:** Mission-critical services experience minimal disruption even during infrastructure events.
+
+## Pricing
+
+AKS Automatic pricing includes a fixed monthly cluster fee and per-vCPU charges on top of standard VM compute costs. This pricing includes financially backed SLAs for both API server uptime and pod readiness. For current rates and a full breakdown by VM category, see the [Azure Kubernetes Service pricing page](https://azure.microsoft.com/pricing/details/kubernetes-service#pricing).
+
+## Getting started
+
+### Prerequisites
+
+- Azure CLI 2.77.0 or later.
+- `aks-preview` extension 19.0.0b15 or later.
+
+```bash
+# Install or update the aks-preview extension
+az extension add --name aks-preview
+az extension update --name aks-preview
+```
+
+### Register the preview feature
+
+```bash
+az feature register --name AKS-AutomaticHostedSystemProfilePreview --namespace Microsoft.ContainerService
+```
+
+### Create the cluster
+
+Select a region where managed system node pools are available. See the [regional availability](https://aka.ms/aks/automatic/managed-systempool-regions) section in the documentation for the current list.
+
+#### Set your variables
+
+```bash
+RESOURCE_GROUP="myResourceGroup"
+CLUSTER_NAME="myAKSCluster"
+LOCATION="westcentralus"  # Choose a supported region (see: https://aka.ms/aks/automatic/managed-systempool-regions)
+```
+
+#### Create the resource group
+
+```bash
+az group create --name $RESOURCE_GROUP --location $LOCATION
+```
+
+#### Create an Automatic cluster with a managed system node pool
+
+```bash
+az aks create \
+--resource-group $RESOURCE_GROUP \
+--name $CLUSTER_NAME \
+--location $LOCATION \
+--sku automatic \
+--enable-hosted-system
+```
+
+The output includes `"hostedSystemProfile": { "enabled": true }` confirming the feature is active.
+
+### Connect to the cluster and deploy an application

```

Comment:
What kind of applications will benefit from managed system pool  & AKS Automatic ? can we list based on the persona of customers considering AKS Automatic and possibly by verticals - healthcare, gaming, ecommerce etc. Those tangible examples of why they should use it and what benefit they derive for their workloads & operational time/ cost save should be talked about in a section.

Reply:
I don't think there's a vertical play. Anything else will be a reiteration of what was said at the beginning of the article?